main.py

```diff
@@ -1,10 +1,42 @@
 import init_django_orm  # noqa: F401
 
+
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player, player_data in players.items():
+        race, created = Race.objects.get_or_create(
+            name=player_data["race"]["name"],
+            description=player_data["race"]["description"]
+        )
+        if player_data["guild"]:
```

Comment:
It is recommended to use the .get() method instead of [] because it is more readable and safer if there are no objects in the dictionary with the key you want to retrieve.